SimpleBlockX4.md

```diff
@@ -0,0 +1,131 @@
+      * In command line type `subtract vol 1 2 3 4 from 5 keep` or `subtract vol 1 to 4 from 5 keep`. Then type `delete vol 5`.
+      * Now you are left with a brick void volume with cutouts around the inner volumes and a graveyard volume surrounding everything else.
+
+#### Imprinting and Merging Volumes in Cubit/Trelis
+_Imprinting and Merging surfaces accelerates the DAGMC process by combining surfaces and simplifying the process so that no area in space can be owned by more than one volume. The default merge tolerance for distance between surfaces is 5.0e-4._
+
+  1. Imprint and Merge all shared surfaces in the model.
+      * In command panel select __Geometry -> Volume -> Imprint Merge__ and select __Imprint and Merge__ from the dropdown menu. Then type "all" into the __Volume ID(s)__ box and click __Apply__.
+      * In command line type `imprint volume all` and then enter. Then type `merge volume all`. At times it may be helpful to set the merge tolerance to something other than the default, to do this type "merge tol [new merge tolerance]" before running the merge.
+      * Now all surfaces that previously overlapped have been merged into one surface shared by both volumes.
+
+
+#### Meshing in Cubit/Trelis
+_Meshing is completed automatically when running_ `export dagmc` _, but meshing in Cubit/Trelis can be used to create unstructured mesh tallies. This is an example of how one could use tetmesh to mesh volumes for this purpose. More information on DAGMC unstructured mesh tallies can be found_ [here](https://svalinn.github.io/DAGMC/usersguide/tally.html).
+
+1. Set the sizing factor to prepare part for meshing
+    * In the command panel select the __Mesh (purple cube)-> Volume -> Intervals (square with dimensioning lines)__. Type "all" into the select volumes box to select all of the volumes for setting intervals. Auto factor with the default coarseness is acceptable so hit "Apply".
+    * For command line type `volume all size auto factor 5`
+      - This command creates an auto sized mesh with the default/medium coarseness
+      - Typing `volume all size #` would select the first four volumes as listed in the model tree and set the size of any future mesh to # units.
+    * Note that one could select smaller subsets of volumes and mesh each at different levels of coarseness. This may be useful if some volumes require more/less precision than others.
+2. Mesh files with tetmesh
+    * In the command panel directly to the right of the "Intervals" action select the "Mesh" action. Type `all` into the box under "Select Volumes" or click the box and hold the control key while selecting all of the cubes. Then select "Tetmesh" from the dropdown menu under "Select Meshing Scheme" and click mesh.

```

Comment:
formatting: use the same bold font like you do everywhere else to indicate selecting things in the GUI